daq/forchestrator.py

```diff
@@ -40,15 +42,27 @@ def _handle_faucet_events(self):
             LOGGER.debug('Faucet event %s', event)
             if not event:
                 return True
+
+            timestamp = event.get("timestamp", time.time())
+
             (dpid, port, active) = self._faucet_events.as_port_state(event)
             if dpid and port:
                 LOGGER.info('Port state %s %s %s', dpid, port, active)
+                port_state_event = faucet_events.PortStateEvent(dpid, timestamp, port, active)
```

Comment:
these two lines really should just be:

faucet_state.port_state_update(timestamp, dpid, port, active)

where faucet_state is an instance of a state_collector class (formerly faucet_events)